src/metabase/api/embed.clj

```diff
@@ -151,19 +152,19 @@
   [card]
   (update card :parameters concat (template-tag-parameters card)))
 
-(s/defn ^:private apply-merged-id->value :- (s/maybe [{:slug   su/NonBlankString
-                                                       :type   s/Keyword
-                                                       :target s/Any
-                                                       :value  s/Any}])
-  "Adds `value` to parameters with `slug` matching a key in `merged-id->value` and removes parameters without a
+(s/defn ^:private apply-slug->value :- (s/maybe [{:slug   su/NonBlankString
+                                                  :type   s/Keyword
+                                                  :target s/Any
+                                                  :value  s/Any}])
+  "Adds `value` to parameters with `slug` matching a key in `merged-slug->value` and removes parameters without a
```

Comment:
Minor correctness tweaks. These functions actually are passed a map of parameter _slug_ to value, so it's super misleading that they all have names like `id->value`... fixed that.